### DISCUSSION TITLE
作者好，1.6版貌似有bug，1.5版没问题

### DISCUSSION THREAD

**Discussion by ghost** [LeetaoGoooo/RSSAid#22] [Q&A] [answered]
作者你好，我的手机是荣耀和诺基亚的可以安装1.6版的各种app包，但是打开后进入的都是灰白色没有任何文字的页面，最后只能退出，不知道是哪里的问题。
另外1.5版没问题，可以正常进入页面。

**Comment by LeetaoGoooo** [comment, marked as answer]
> 作者你好，我的手机是荣耀和诺基亚的可以安装1.6版的各种app包，但是打开后进入的都是灰白色没有任何文字的页面，最后只能退出，不知道是哪里的问题。
> 另外1.5版没问题，可以正常进入页面。

你重新下载一下 1.6.0 版本的～刚才我又重新发布了一下，修复了你说的这个问题

**Reply by ghost** [reply]
完美解决，谢谢。
